lib/MetaCPAN/Role/HasConfig.pm

```diff
@@ -24,13 +25,12 @@ sub _build_config {
     my $config = $self->_zomg("$FindBin::RealBin/..");
     return $config if $config;
 
-    require_module('Git::Helpers');
-    $config = $self->_zomg( Git::Helpers::checkout_root() );
+    $config = $self->_zomg( checkout_root() );
```

Comment:
This removes the `require` on `Git::Helpers`, but still uses the `checkout_root()` function, so we'll need to add it to the `use` statements above.

Reply:
Oh, I see it has moved to `MetaCPAN::Util`.